modules/account-lib/src/index.ts

```diff
@@ -92,6 +92,9 @@ export { Polygon };
 import * as Sui from '@bitgo/sdk-coin-sui';
 export { Sui };
 
+import * as Cosmos from '@bitgo/abstract-cosmos';
```

Comment:
I think we should export classes only from coin sdks.
If you think there is something like iface not exported in sdk-coin, we can export it there and use it from that.

This will help in backward compatibility and consistency and WP not dependent on abstract-cosmos. As abstract-cosmos is just facilitating abstraction for cosmos side chains.

Reply:
+1 That's the ideal approach. I'm okay with this as long as we can sufficiently abstract tx building process. I'll try to export what we need via the individual coins only and see if works as intended. Will fall back to this approach if coin specific export doesn't help.